central/notifications/datastore/datastore_test.go

```diff
@@ -0,0 +1,168 @@
+package datastore
```

Comment:
The file should be named `datastore_impl_test.go`, according to go convetions the current test file naming covers `datastore.go`, which it really doesn't.